torch/ao/quantization/experimental/fake_quantize.py

```diff
@@ -0,0 +1,35 @@
+import torch
+from torch import Tensor
+from torch.ao.quantization.experimental.observer import APoTObserver
+from torch.ao.quantization.experimental.quantizer import quantize_APoT, dequantize_APoT
+from torch.ao.quantization.fake_quantize import FakeQuantizeBase
+
+class APoTFakeQuantize(FakeQuantizeBase):
+    alpha: Tensor
+    gamma: Tensor
+    quantization_levels: Tensor
+    level_indices: Tensor
+
+    def __init__(self, observer: APoTObserver):
+        super().__init__()
+        self.activation_post_process = observer
```

Comment:
need constructor args?

Reply:
yeah, original code is probably a bit confusing, maybe we can rename `observer` to `observer_cls` or `observer_constructor` to make it clearer, we do need to instantiate the observer in `__init__.py`: https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fake_quantize.py#L133